resources/js/Components/Skeleton/Skeleton.tsx

```diff
@@ -7,10 +7,12 @@ interface SkeletonProperties {
     height?: string | number;
     isCircle?: boolean;
     className?: string;
+    disabled?: boolean;
```

Comment:
maybe we should rename this `animationDisabled` so it's clear what disabling does? 🤔